serverlessworkflow/modules/ROOT/pages/cloud/deploying-on-minikube.adoc

```diff
@@ -1 +1,369 @@
+----
+
+Now with all the prerequisites in place, we can go ahead and deploy our Serverless Workflow Application on Minikube.
+
+First, let's create a new namespace and switch to that context.
+
+.Procedure
+. Create the `serverless-workflow-greeting-quarkus` namespace
++
+[source,shell]
+----
+kubectl create namespace serverless-workflow-greeting-quarkus
+----
+
+. Set the context to the newly created namespace:
++
+[source,shell]
+----
+kubectl config set-context --current --namespace=serverless-workflow-greeting-quarkus
+----
++
+Now on, all the commands issued against Minikube, will use this namespace
++
+[NOTE]
+====
+To check the current namespace, the following command can be used.
+
+[source,shell]
+----
+kubectl config view --minify -o jsonpath='{..namespace}'
+----
+====
+
+Let's proceed with the installation of our Serverless Application using the container generated in the xref:cloud/build-workflow-image-with-quarkus-cli.adoc[Building Serverless Workflow Images using Quarkus CLI] guide. For this example we will use the `native` image to take advantage of its faster startup. Below you can find two examples, one using the Knative CLI `kn` and another using the `kubectl` command line tool.
+
+
+=== Deploying with Knative CLI
+
+[source,shell]
+----
+kn service create hello \
+    --image=dev.local/kogito/serverless-workflow-greeting-quarkus:1.0-native \
+    --pull-policy=IfNotPresent \
+    --port 8080
+----
+As output, you should have something similar to the following:
+[source,shell]
+----
+Creating service 'hello' in namespace 'serverless-workflow-greeting-quarkus':
+
+  0.066s The Route is still working to reflect the latest desired specification.
+  0.099s ...
+  0.322s Configuration "hello" is waiting for a Revision to become ready.
+  4.885s ...
+  5.061s Ingress has not yet been reconciled.
+  5.322s Waiting for load balancer to be ready
+  5.460s Ready to serve.
+
+Service 'hello' created to latest revision 'hello-00001' is available at URL:
+http://hello.serverless-workflow-greeting-quarkus.10.103.94.37.sslip.io
+----
+
+You can now use the url shown on the command output to call the Serverless Workflow:
+[source,shell]
+----
+curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"workflowdata" : {"name": "John", "language": "English"}}' http://hello.serverless-workflow-greeting-quarkus.10.103.94.37.sslip.io/jsongreet
+----
+
+As response, you should get something like:
+[source,json]
+----
+{"id":"0f77abce-837e-4bd2-b4f1-a0e5e0265fcb","workflowdata":{"name":"John","language":"English","greeting":"Hello from JSON Workflow, "}}%
+----
+
+
+=== Deploying with Kubectl
+
+You can also use plain Kubernetes objects to deploy Serverless Workflow.
+Instead creating the `knative` 'yaml|json' descriptors, you can leverage on the Quarkus Kubernetes extension and Kogito knative addon to generate that for you. To use this, make sure the following dependencies are present on the Application dependencies:
```

Comment:
```suggestion
Instead of creating the `knative` 'yaml|json' descriptors, you can leverage the Quarkus Kubernetes extension and Kogito Knative Addon to generate that for you. To use this, ensure the following dependencies are present on the Application dependencies:
```